cmd/up.go

```diff
@@ -425,6 +425,8 @@ func (cmd *UpCmd) devPodUpProxy(
 		baseOptions.IDE = workspace.IDE.Name
 		baseOptions.IDEOptions = nil
 		baseOptions.Source = workspace.Source.String()
+		baseOptions.GitOverrideUsername = cmd.GitUsername
```

Comment:
Not sure if we need this here. This will be executed on the dev machine and they won't have the username and token flag set

Reply:
this is more for testing, since those args are hidden anyway. can remove though if you think it's misleading

Reply:
it's a bit unintuitive when reading through it but `devPodUpProxy` will be executed on the devs machine, initiating the websocket connection and `devPodUpMachine` will then be run on the remote side

Reply:
ah 🤦  kk thx on it!